tests/test_model_unicode.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+import unittest
+
+from tests.setup_database import Setup
+from tests.utils import OpenEventTestCase
+
+from open_event import current_app as app
+from open_event.helpers.data import save_to_db
+from open_event.models.event import Event
+from open_event.models.session import Session, Level, Format, Language
+from open_event.models.speaker import Speaker
+from open_event.models.sponsor import Sponsor
+from open_event.models.microlocation import Microlocation
+
+UNICODE_STRING = u'☺'
+
+
+class TestModelUnicode(OpenEventTestCase):
+    """
+    Tests for unicode handling in models
+    """
+    def setUp(self):
+        self.app = Setup.create_app()
+        with app.test_request_context():
+            event = Event(name=UNICODE_STRING,
+                          start_time=datetime(2013, 8, 4, 12, 30, 45),
+                          end_time=datetime(2016, 9, 4, 12, 30, 45))
+            event.owner = 1
+
+            microlocation = Microlocation(name=UNICODE_STRING)
+            level = Level(name=UNICODE_STRING, event_id=1)
+            format = Format(name=UNICODE_STRING, label_en='label', event_id=1)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'format'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2572200721/issues/source?bid=3233864&fileBranchId=3298329#l33)